ci/tasks/deploy.sh

```diff
@@ -82,8 +78,7 @@ resource_pools:
 - name: vms
   network: private
   stemcell:
-    url: $STEMCELL_URL
-    sha1: $STEMCELL_SHA
+    url: file://stemcell.tgz
```

Comment:
@AbelHu just an FYI if you decide to use the stemcell fetched by the resource from the s3 bucket, you'll need to upload a new stemcell to this bucket anytime its changed. Since concourse doesn't really have a resource to fetch contents from an arbitrary url and doesn't have an azure storage resource, we opted to specify the stemcell from the azure storage link, rather than use the stemcell that is duplicated in s3.